src/core/client/stream/tabs/Comments/ReplyList/ReplyList.tsx

```diff
@@ -102,6 +100,7 @@ const ReplyList: FunctionComponent<ReplyListProps> = (props) => {
               fullWidth
               data-key-stop
               data-is-load-more
+              data-is-view-new
```

Comment:
Was this intentional? I only ask as the only other changes here are for tombstone-ing. Completely fine if it is, but I'm checking this isn't a copy-paste flub since this is a `ReplyList` that I believe is a `data-is-load-more` versus a `data-is-view-new`.